stories/2023-02-01-Overwatch-League/OWL_story.ipynb

```diff
@@ -0,0 +1,3821 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.        with out:
</pre><p>maybe worth including a link to the IPython widgets docs (because the way the widget is used is a bit unfamiliar on its own)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/TuringDataStories/pull/218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
Added